notifications/rest/notifications/send-notification-detailed/send-notification-detailed.3.x.js

```diff
@@ -11,23 +11,11 @@ var service = client.notify.services('ISxxx');
 
 service.notifications
   .create({
-    "identity": "00000001",
-    "title": "Generic loooooooong title for all Bindings",
-    "body": "This is the body for all Bindings",
-    "fcm": {
-      "notification": {
-        "title": "New alert",
-        "body": "Hello Bob!"
-      }
-    },
-    "apn": {
-      "aps": {
-        "alert": {
-          "title": "New alert",
-          "body": "Hello Bob!"
-        }
-      }
-    },
-  }).then(function(notification) {
+    identity: '00000001',
+    title: 'Generic loooooooong title for all Bindings',
+    body: 'This is the body for all Bindings',
+    fcm: '{"notification":{"title":"New alert","body":"Hello Bob!"}}',
+    apn: '{"aps":{"alert":{"title":"New alert.","body":"Hello Bob!"}}}'
```

Comment:
@hortega this file was updated before to use the creation this way 
[this is the PR](https://github.com/TwilioDevEd/api-snippets/commits/3bb55ac9c71365015474f81efffaaf90f744171e/notifications/rest/notifications/send-notification-detailed/send-notification-detailed.3.x.js?author=well1791), but if the CI get errors because of this, we should report it.